runtime/src/lib.rs

```diff
@@ -88,11 +94,9 @@ pub mod opaque {
 	/// Opaque block identifier type.
 	pub type BlockId = generic::BlockId<Block>;
 
+	//TODO Does this need to remain at all?
```

Comment:
i dont think it does

Reply:
I couldn't figure out how to remove it and still get the node to compile :-/